osquery/tables/applications/browser_utils.cpp

```diff
@@ -103,7 +118,12 @@ void genExtension(const std::string& uid,
 
   r["identifier"] = fs::path(path).parent_path().parent_path().leaf().string();
   r["path"] = path;
-  results.push_back(r);
+
+  // Emit one row for each permission we have
```

Comment:
It definitely the considerable idea. What do you think about storing all the permissions together separated somehow? I think, we have this kind of practice in some other tables. Will do my homework about that.  :)

Reply:
I did consider saving them all in a single row, but we do have tables that emit multiple rows in order to simplify filtering at query time, so I ended up using this approach.